sourcegraph/sourcegraph
# licensing: implement cody landing page

Closes #59693

On a cody-only license, enterprise users won't have access to Cody, so we provide them with a landing page for Cody to upsell the product.

![CleanShot 2024-01-30 at 21 48 18](https://github.com/sourcegraph/sourcegraph/assets/25608335/db9c229f-b0bf-4584-8797-a66d14d3c6c1)

## Test plan

* Start up a Sourcegraph instance with a Code Search only license.
* Navigate to the home page
* Clicking on the `Cody Ai` link on the Navbar should show a landing page for upselling Cody.
